jujugui/static/gui/src/app/components/generic-button/generic-button.js

```diff
@@ -35,6 +36,7 @@ YUI.add('generic-button', function() {
     _generateClasses: function() {
       return classNames(
         this.props.type ? 'button--' + this.props.type : 'button--neutral',
+        this.props.classNames,
```

Comment:
I'm not a huge fan of adding more class names to the buttons in this way. You could use the type prop or add a wrapping element.